packages/flutter/test/cupertino/route_test.dart

```diff
@@ -1052,6 +1055,38 @@ void main() {
     expect(rootObserver.dialogCount, 0);
     expect(nestedObserver.dialogCount, 1);
   });
+
+  testWidgets('showCupertinoModalPopup allows for semantics dismiss on iOS', (WidgetTester tester) async {
+    debugDefaultTargetPlatformOverride = TargetPlatform.iOS;
+    final SemanticsTester semantics = SemanticsTester(tester);
+    await tester.pumpWidget(CupertinoApp(
+      home: Navigator(
+        onGenerateRoute: (RouteSettings settings) {
+          return PageRouteBuilder<dynamic>(
+            pageBuilder: (BuildContext context, Animation<double> _, Animation<double> __) {
+              return GestureDetector(
+                onTap: () async {
+                  await showCupertinoModalPopup<void>(
+                    context: context,
+                    builder: (BuildContext context) => const SizedBox(),
+                  );
+                },
+                child: const Text('tap'),
+              );
+            },
+          );
+        },
+      ),
+    ));
+
+    // Push the route.
+    await tester.tap(find.text('tap'));
+    await tester.pumpAndSettle();
+
+    expect(semantics, includesNodeWith(actions: <SemanticsAction>[SemanticsAction.tap]));
```

Comment:
put this into one expect:

```dart
includesNodeWith(actions: <SemanticsAction>[SemanticsAction.tap], label: 'Dismiss')
```

You want to ensure that there is a single node with tap action _and_ dismiss label.

Your current code only verifies that there is a node with tap action and potentially a different node that has the dismiss label.

Reply:
Good point, fixed!